parkingspaces/src/main/java/com/baspiotr/parkingspaces/domain/utils/Fee.java

```diff
@@ -0,0 +1,44 @@
+package com.baspiotr.parkingspaces.domain.utils;
+
+import com.baspiotr.parkingspaces.domain.model.Role;
+
+public class Fee {
+
+    private static double feeRegular(double n) {
+        if (n == 0)
```

Comment:
code style - never use this kind of variables like n, I have no idea what is that

Reply:
it should always reflect what it is